src/plugins/expressions/common/execution/execution.test.ts

```diff
@@ -236,6 +236,40 @@ describe('Execution', () => {
       expect(typeof result).toBe('object');
       expect((result as AbortSignal).aborted).toBe(false);
     });
+
+    test('can abort an expression', async () => {
+      const execution = createExecution('sleep 10');
+      execution.start();
+
+      execution.cancel();
+      const result = await execution.result;
+
+      expect(result).toMatchObject({
+        type: 'error',
+        error: {
+          message: 'The expression was aborted.',
+          name: 'AbortError',
+        },
+      });
+    });
+
+    test('can abort an expression which has function running mid flight', async () => {
+      const execution = createExecution('sleep 300');
```

Comment:
Not a big deal here (since you're only waiting 1ms) but it might not be a bad idea to use fake timers in this scenario (in case for some reason this test breaks in the future, so you don't end up actually waiting 300ms).

Reply:
Done.